SERVER/static/js/chat.js

```diff
@@ -21,20 +21,36 @@
   };
 
   sendMessage = function() {
-    var innerText;
-    innerText = $.trim($("#text").val());
-    var packet = new Object();
-    packet.username = "Adam";
-    packet.message = innerText;
-    packet.date = new Date().toUTCString();
-    var myPacket = JSON.stringify(packet);
-    console.log(myPacket);
-    if (innerText !== "") {
-      $(".messages").append("<li class=\"i\"><div class=\"head\"><span class=\"time\">"+new Date().toUTCString()+"</span><span class=\"name\"></span></div><div class=\"message\">"+innerText+"</div></li>");
+    var innerText = $.trim($('#text').val());
+    if (innerText !== ""){
+    var packet = {'username':'Adam' , 'message':innerText , 'date':new Date().toUTCString()};
+    $.ajax({
+        url:'/receive_message',
+        data: JSON.stringify(packet),
+        type: 'POST',
+        contentType: 'application/json',
+        dataType: 'json',
+        success: function(msg){
+            console.log("Good news every one ", msg);
+        }
+    });
+      getMessage();
+      $('.messages').append("<li class=\"i\"><div class=\"head\"><span class=\"time\">"+new Date().toUTCString()+"</span><span class=\"name\"></span></div><div class=\"message\">"+innerText+"</div></li>");
       clearResizeScroll();
-
     }
   };
+
+  getMessage = function(){
```

Comment:
As stated in my PR no this does not. Since im getting a varying array back from the server full of null things. I need something that is more consistent from the server that way i can update consistently everytime. Currently it still posts to the message box as it always has.